tests/framework/extensions/rbac/constants.go

```diff
@@ -0,0 +1,46 @@
+package rbac
+
+const (
+	Admin                            = "admin"
+	RestrictedAdmin                  = "restricted-admin"
+	StandardUser                     = "user"
+	RoleReadOnly                     = "read-only"
+	RoleOwner                        = "cluster-owner"
+	RoleMember                       = "cluster-member"
+	RoleProjectOwner                 = "project-owner"
+	RoleProjectMember                = "project-member"
+	RoleCrtbView                     = "clusterroletemplatebindings-view"
+	RolePrtbManage                   = "projectroletemplatebindings-manage"
+	RoleProjectsCreate               = "projects-create"
+	RoleProjectsView                 = "projects-view"
+	RoleCreateNS                     = "create-ns"
+	RoleCustomManageProjectMember    = "projectroletemplatebindings-manage"
+	RoleCustomCreateNS               = "create-ns"
+	RoleProjectReadOnly              = "read-only"
+	DefaultNamespace                 = "fleet-default"
+	RancherNamespace                 = "cattle-system"
+	KubeSystemNamespace              = "kube-system"
+	LocalCluster                     = "local"
+	Projects                         = "projects"
+	ProjectRoleTemplateBindings      = "projectroletemplatebindings"
+	ClusterRoleTemplateBindings      = "clusterroletemplatebindings"
+	Roles                            = "roles"
+	ClusterRoles                     = "clusterroles"
+	RoleBindings                     = "rolebindings"
+	ClusterRoleBindings              = "clusterrolebindings"
+	GlobalRoles                      = "globalroles"
+	GlobalRoleBindings               = "globalrolebindings"
+	RoleTemplates                    = "roletemplates"
+	RancherConfigMap                 = "cattle-controllers"
+	RancherLeaderAnnotation          = "control-plane.alpha.kubernetes.io/leader"
+	ProjectIDAnnotation              = "field.cattle.io/projectId"
+	CrtbGrbOwnerLabel                = "authz.management.cattle.io/grb-owner"
+	MemberBindingLabel               = "membership-binding-owner"
+	ProjectScopedLabel               = "cattle.io/project-scoped"
+	ProjectScopedLabelValue          = "original"
+	ProjectScopedNamespaceLabelValue = "copy"
+	NormanCreatorLabel               = "cattle.io/creator"
+	NormanCreatorLabelValue          = "norman"
+	ActiveStatus                     = "active"
+	ForbiddenError                   = "403 Forbidden"
+)
```

Comment:
Looks like below constants are Not used, please remove
RoleCrtbView
RolePrtbManage
RoleProjectsCreate
RoleProjectsView
RoleCreateNS
RancherNamespace
KubeSystemNamespace
LocalCluster
RancherConfigMap
RancherLeaderAnnotation
ProjectIDAnnotation
CrtbGrbOwnerLabel
MemberBindingLabel
ProjectScopedLabel
ProjectScopedLabelValue
ProjectScopedNamespaceLabelValue
NormanCreatorLabel
NormanCreatorLabelValue

Reply:
agreed. these changes are needed as part of @Priyashetty17's PR. we can update these as part of that PR instead.